crmsh/ui_resource.py

```diff
@@ -218,6 +218,7 @@ class RscMgmt(command.UI):
     }
     rsc_failcount = {
         'set': "crm_attribute -t status -n 'fail-count-%s' -N '%s' -v '%s' -d 0",
+        'set_p': "crm_attribute -t status -P 'fail-count-%s' -N '%s' -v '%s' -d 0",
```

Comment:
`-d <value>` is meaningful only when being used with "--query/-G" though, meaning if there's no such an entry in cib, the caller of this command gives himself a `default_value` defined by himself rather than getting an error.

Reply:
OK, I've removed `-d` for both 'set' and 'set_p'